File: DataStructures/LinkedList/IterativeLength.js
class Node {
    constructor(val)
    {
        this.data = val;
        this.next = null;
    }
}
var head;

function push(new_data)
{
    var new_node = new Node(new_data);
    new_node.next = head; 
    head = new_node;
}

function getCount() {
    var temp = head;
    var count = 0;
    while(temp != null)
    {
        count++;
        temp = temp.next;
    }
    return count;
}

push(1);
push(5);
push(4);
push(3);
push(2);

document.write("COunt of nodes is + " + getCount());

File: DataStructures/LinkedList/IterativeLength.cpp
#include <cstdlib>

using namespace std;

class Node 
{
    public: 
    int data;
    Node * next;
};


void push(Node** head_ref, int new_data)
{
    Node * new_node = new Node();
    new_node->data = new_data;
    new_node->next = *head_ref;
    *head_ref = new_node;
}

int getCount(Node* head)
{
   
    int count = 0;
    Node * current = head;
    while(temp != nullptr)
    {
        current = current->next;
        count = count + 1;
    }
    return count;
}

int main()
{
    Node* head = nullptr;

    push(&head, 2);
    push(&head, 5);
    push(&head, 4);
    push(&head, 3);
    cout << "count of nodes is " << getCount(head);
    return 0;
}